### PR TITLE
Awesome books classes

### PR DESCRIPTION
**Pull request by MostafaShahat2021**
### In this project we :
- Using Classes instead of Objects.
- Add Clear Input field Function.
- Update page styling.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,13 +12,13 @@
 <body>
     <h1>Awesome books</h1>
    <section class="container">
-   </section>
-    <hr>
+</section>
```
Comment (ShadyShawkat) [inline index.html:15]: - Please make sure to stick to the design as much as possible, which includes:
   - [x] Adjusting the spacing between the list elements to better match the original wireframe.
   - [x] Editing the background of the book list to alternate between dark and light backgrounds.

| Your design | Original design |
| --- | --- |
| image | image |

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi Team, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- All linters are passing ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (MostafaShahat2021) [reply index.html:15]: @ShadyShawkat Thanks a lot for your efforts in reviewing our code

**Review by prolajumokeoni — APPROVED** [review]
Hi @MostafaShahat2021 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Git flow was used correctly 🚀 
- Good Pr title and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by MostafaShahat2021** [comment]
Hi @prolajumokeoni, Thanks a lot for your efforts in reviewing our code. 😄

**Pull request merged**